src/it/strip-tar/build1.properties

```diff
@@ -0,0 +1,2 @@
+# current tar archive strip implementation does not 'fix' content under META-INF
+# user.name=user1
```

Comment:
If we only put non-unpacked JAR files in the tar archive then we don't need to do anything special. Anyway, you should not comment this line, because you should have the same archive whatever your user name is. If not, the archive is not reproducible.

Reply:
@Zlika AFAIU that goes in the manifest, which we don't clean up for tar archives. So, if we uncomment that line then we need to strip manifests as well or the IT will fail. I'm not an expert on the topic though so I'll follow your advice :-)

Reply:
@Zlika I think I just understood what you meant here, sorry I'm proficient at mvn, but not as expert as you :-) Let me try to fix this